spec/aggregate_root_spec.rb

```diff
@@ -3,16 +3,18 @@
 class Order
   include RailsEventStore::AggregateRoot
 
+  attr_accessor :id, :status
```

Comment:
Those fields were already public. I think it was to be able to assert status value and infer that even was properly applied. While I agree that in real context they should be private, is it that important here?